http-api/src/issues.rs

```diff
@@ -68,10 +77,13 @@ async fn issue_handler(
     let storage = ctx.storage().await?;
     let whoami = person::local(&*storage).map_err(Error::LocalIdentity)?;
     let issues = issue::Issues::new(whoami, &ctx.paths, &storage).map_err(Error::Issues)?;
-    let issue = issues
+    let mut issue = issues
         .get(&project, &issue_id)
         .map_err(Error::Issues)?
         .ok_or(Error::NotFound)?;
+    issue
+        .resolve(storage.as_ref())
+        .map_err(Error::IdentityResolveError)?;
```

Comment:
Do we want to bail here? I'm not sure. We should still be able to render the issue without a resolved author I would think?

Reply:
No you're right, we should just warn about it, for debugging purposes